File: src/main/java/it/contrader/dto/ContentDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class ContentDTO {

	private Long id;

	private String tag;

	private String title;

	private String text;
	
	private int idTraining;

}

File: src/dto/contentdto.ts
import {TrainingDTO} from './trainingdto';
import { LoginComponent } from 'src/app/login/login.component';

export class ContentDTO{
    id:number;

    tag:string;
    text:string;
    title:string;
    idTraining:number;

    training: TrainingDTO;

    
}